package.json

```diff
@@ -38,5 +39,12 @@
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "typescript": "5.2.2"
+  },
```

Comment:
i dont think this is needed. if you want to change the coverage path in the above file